src/aws_sdk_dynamodbstore.rs

```diff
@@ -25,19 +25,19 @@ pub struct Backend {
 const NO_SORT_KEY: &str = "_";
 
 fn new_item<'h, 's, S: Into<Arg<'s>> + Send, A: IntoIterator<Item = (&'static str, AttributeValue)>>(
-    hash: ExplicitKey<'h>,
+    hash: &ExplicitKey<'h>,
     sort: S,
     attrs: A,
 ) -> HashMap<String, AttributeValue> {
     let mut ret: HashMap<_, _> = attrs.into_iter().map(|(k, v)| (k.to_string(), v)).collect();
-    ret.insert("hk".to_string(), attribute_value(hash.unredacted));
+    ret.insert("hk".to_string(), attribute_value(Arg::Borrowed(hash.unredacted.as_bytes())));
```

Comment:
`hash.unredacted.as_bytes()` is already a reference to the underlying data, why need to wrap it with `Arg::Borrowed`?

Reply:
Thanks for catching!

Reply:
Removed. I was thinking `attribute_value` takes an `Arg`. But it actually takes an `Into<Arg>`, so yeah, this wrapping happens automatically.